lua/entities/gmod_wire_expression2/core/string.lua

```diff
@@ -36,6 +36,76 @@ end)
 
 /******************************************************************************/
 
+registerOperator("fea", "nss", "", function(self, args)
+	local keyname, valname = args[2], args[3]
+	local sub = utf8.sub
+	local str = args[4]
+	str = str[1](self, str)
+
+	local statement = args[5]
+
+	for key=1, #str do
```

Comment:
If you are going to make this utf8 compliant it should use utf8.len instead of #str